File: v6/sprefa-extract/tests/fixtures/ts/scip/alpha.ts
// scip-ratchet fixture (4c-ii): one of TWO corpus defs named `helper`, so the
// v5-shaped name-match is AMBIGUOUS corpus-wide and binds nothing. scip (the
// real compiler) resolves gamma.ts's call HERE through the import.
export function helper(): number {
  return 1;
}

File: v6/sprefa-extract/tests/fixtures/go/scip/alpha/alpha.go
// Package alpha is one of two same-name Helper providers, so the AST
// name-match across the corpus is ambiguous and scip's import-aware
// resolution is what settles a call through the import (ScipOverride).
package alpha

// Helper answers with the alpha marker.
func Helper() string {
	return "alpha"
}
